yb-voyager/cmd/exportData.go

```diff
@@ -547,7 +548,14 @@ func getFinalTableColumnList() (map[string]string, []sqlname.NameTuple, *utils.S
 	tableListFromDB := source.DB().GetAllTableNames()
 	var err error
 	var fullTableList []sqlname.NameTuple
+	logMiningFlushTable := utils.GetLogMiningFlushTableName(migrationUUID)
 	for _, t := range tableListFromDB {
+		if source.DBType == ORACLE {
```

Comment:
How about adding this logic to `GetAllTableNames` and `GetAllTableNamesRaw` of the Oracle implementation of source db? That way it will get filtered out of everywhere..

We probably don't need exact match, we can check for prefix..? (since migrationUUID won't be available there.)

Reply:
yeah, if an exact match is not required then its better in `GetAllTableNamesRaw`.